### PR TITLE
style: update coding style by php-cs-fixer v3.42.0

### PR DESCRIPTION
**Pull request by kenjis**
**Description**
Update by php-cs-fixer v3.42.0

**Checklist:**
- [x] Securely signed commits
- [] Component(s) with PHPDoc blocks, only if necessary or adds value
- [] Unit testing, with >80% coverage
- [] User guide updated
- [x] Conforms to style guide

### CODE REVIEW

**Comment by kenjis** [comment]
Rector adds unneeded `@see` tags...

```
1) system/Commands/Database/MigrateStatus.php:18

    ---------- begin diff ----------
@@ @@
  * Displays a list of all migrations and whether they've been run or not.
  *
  * @see MigrateStatusTest
+ * @see \CodeIgniter\Commands\Database\MigrateStatusTest
  */
 class MigrateStatus extends BaseCommand
 {
    ----------- end diff -----------

Applied rules:
 * AddSeeTestAnnotationRector
 ```
https://github.com/codeigniter4/CodeIgniter4/actions/runs/7320300308/job/19939228674?pr=8367

**Comment by kenjis** [comment]
@sammyskills Do you think https://github.com/codeigniter4/CodeIgniter4/pull/8367#issuecomment-1868853441 is a bug in AddSeeTestAnnotationRector?

**Comment by sammyskills** [comment]
No, I don't think so. 

From the docs:
>Make it FQN, so it stays in the annotation, not in the PHP source code.

So, the FQN annotation should be used instead.

**Comment by kenjis** [comment]
@sammyskills Sorry, I was confused. I mean @samsonasik See https://github.com/codeigniter4/CodeIgniter4/pull/8367#issuecomment-1868869085

**Comment by samsonasik** [comment]
i am on mobile, iirc, `@see` is generic tag iirc, so that value is detected as text so that expected, i think fqcn should be used.

**Comment by kenjis** [comment]
@sammyskills @samsonasik Thank you.
I sent a bug report https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/issues/7620

File: system/ThirdParty/Escaper/Escaper.php [hunk 1]
```diff
@@ -157,9 +157,21 @@ public function __construct(?string $encoding = null)
...
+        $this->jsMatcher       =
+            /** @param array<array-key, string> $matches */
+            function (array $matches): string {
+                return $this->jsMatcher($matches);
+            };
+        $this->cssMatcher      =
+            /** @param array<array-key, string> $matches */
+            function (array $matches): string {
+                return $this->cssMatcher($matches);
+            };
```
Comment (kenjis) [inline system/ThirdParty/Escaper/Escaper.php:174]: Oh, I accidentally add these changes...

**Pull request closed without merging**